x/perp/v2/keeper/hooks.go

```diff
@@ -50,6 +50,12 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, _ uint64)
 		// See https://www.notion.so/nibiru/Funding-Payments-5032d0f8ed164096808354296d43e1fa for an explanation of these terms.
 		premiumFraction := markTwap.Sub(indexTwap).QuoInt64(int64(intervalsPerDay))
 
+		if premiumFraction.GT(market.MaxPremiumFraction) {
+			premiumFraction = market.MaxPremiumFraction
+		} else if premiumFraction.LT(market.MaxPremiumFraction.Neg()) {
+			premiumFraction = market.MaxPremiumFraction.Neg()
+		}
```

Comment:
Can you put this inside a function so it can be tested in isolation? I.e., 4 cases, pos + over cap, pos + under cap, neg + over cap, and neg + under cap

It would be easier for us to follow in the future than the app keeper tests